validator_client/src/config.rs

```diff
@@ -369,6 +373,16 @@ impl Config {
             );
         }
 
+        if let Some(registration_pubkey_override) =
+            cli_args.value_of("builder-registration-pubkey-override")
+        {
+            config.builder_registration_pubkey_override = Some(
+                registration_pubkey_override
+                    .parse::<PublicKeyBytes>()
+                    .map_err(|_| "builder-pubkey-override is not a valid PublicKeyBytes.")?,
+            );
+        }
```

Comment:
```suggestion
        config.builder_registration_pubkey_override =
            parse_optional(cli_args, "builder-registration-pubkey-override")?;
```

We have a handy little helper function for this. We don't *consistently* use it for legacy reasons, but I think it's worth using for new code.